### PR TITLE
Map to Ensembl gene IDs and CHEBI IDs

### LINKED ISSUE #2: Convert genes to Ensembl Gene IDs

### LINKED ISSUE #3: Map drugs to CHEBI IDs
Look into how to do this - either via [PharmGKB primary data](https://www.pharmgkb.org/downloads), or something like [CHEBI web services](https://www.ebi.ac.uk/chebi/webServices.do).

### PR DESCRIPTION
**Pull request by apriltuesday**
Closes #2, closes #3.

### CODE REVIEW

File: opentargets_pharmgkb/evidence_generation.py [hunk 1]
```diff
@@ -1,39 +1,105 @@
...
+def pipeline(clinical_annot_path, clinical_alleles_path, drugs_path, created_date, output_path):
     clinical_annot_table = pd.read_csv(clinical_annot_path, sep='\t')
     clinical_alleles_table = pd.read_csv(clinical_alleles_path, sep='\t')
+    drugs_table = pd.read_csv(drugs_path, sep='\t')
 
     merged_table = pd.merge(clinical_annot_table, clinical_alleles_table, on='Clinical Annotation ID', how='left')
-    rs_only_table = merged_table[merged_table['Variant/Haplotypes'].str.contains('rs')][pgkb_column_names]
-
+    # Restrict to variants with rsIDs
+    rs_only_table = merged_table[merged_table['Variant/Haplotypes'].str.contains('rs')]
```
Comment (tcezard) [inline opentargets_pharmgkb/evidence_generation.py:37]: We should count how many variant do not have an `rs`

File: opentargets_pharmgkb/evidence_generation.py [hunk 2]
```diff
@@ -1,39 +1,105 @@
...
+        str_to_iri = {
+            s: pool.apply(get_chebi_iri, args=(s,))
+            for s in split_drugs['split_drug'].drop_duplicates().tolist()
+        }
+    mapped_drugs = pd.concat(
+        split_drugs[split_drugs['split_drug'] == s].assign(chebi=none_to_nan(iri))
+        for s, iri in str_to_iri.items()
+    )
+    # Some drugs we can't unambiguously map using OLS, so we rely on primary data provided by PharmGKB.
+    # Using OLS first ensures we get up-to-date IDs wherever possible.
```
Comment (tcezard) [inline opentargets_pharmgkb/evidence_generation.py:92]: We should count drugs that cannot be assigned to CHEBI IRIs.

**Review by tcezard — APPROVED** [review]
Looks good. We should have a subsequent ticket on adding counts.

File: opentargets_pharmgkb/evidence_generation.py (hunk 1 above)
Comment (apriltuesday) [reply opentargets_pharmgkb/evidence_generation.py:37]: created #10, we can add other things to count as we think of them

**Pull request merged**